### DISCUSSION TITLE
cuando intento agregar algunos elementos en collections, no funciona, mensaje "bad request" "refresh"

### DISCUSSION THREAD

**Discussion by z0rro2k** [Cotalker/documentation#402] [4. Bug Reports]
hola
a ver si alguien puede ayudarme

gracias
cuando intento agregar algunos elementos en collections, no funciona, mensaje "bad request" "refresh"

![image](https://user-images.githubusercontent.com/39965925/133319864-177d0806-44ee-49b8-82e6-be56b2f4f47c.png)

**Comment by acedward** [comment]
Si existe otra propiedad, con código "leve" esto fallará.
El código debe ser único.

**Reply by z0rro2k** [reply]
gracias por tu  tiempo para responderme. 

como hago para re activar algunos elementos (codigos) de ciertos collections? he buscado sin exito aun.

otra cosa, si es que pudieran ayudarme ... como se pueden importar valores para cargar los collections? vi una opcion de import pero no la tengo y estoy como admin

de antemano, gracias de nuevo

**Reply by acedward** [reply]
Para reactivar, actualmente sólo se puede a través de la **API** 

https://www.cotalker.com/swagger/core/?key=woubtjf4olr0t4zgutuwn6scbcm6hd3qh1cgl5obmohpbm3mfublnwcvv67lodgjvd3h86s9ppshtvmf95gepsqh6nizq9liu7f#/properties/patchPropertiesId

Pero tu pregunta tiene mucho sentido, y debería existir una forma en la interfaz, voy a levantarlo internamente.

**Reply by acedward** [reply]
Te recomiendo que crees una nueva discusión por cada pregunta, así nada se pierde en el tintero.

**Reply by z0rro2k** [reply]
gracias por responder y ayudarme

**Comment by agallardol** [comment]
@z0rro2k, para ordenar un poco la forma en que te ayudamos, traté de dividir las preguntas:

- Bad request cuando intentas crear el elemento
  - **Aparentemente ese código ya existe y por tanto obtenemos este error. Esta es una incidencia conocida y estamos trabajando para poder validar códigos duplicados previo a intentar crear elementos (o cualquier otro objeto en Cotalker). Esto debería estar disponible en las próximas semanas en nuestra aplicación**
- Imposibildiad de reactivar un elemento desactivado.
  - **Pienso que esto tiene todo el sentido y crearemos un issue para que podamos darle seguimiento. Intentaremos darle prioridad a esta funcionalidad porque imagino que es bloqueante para lo que estás haciendo ahora mismo**
- Cómo hacemos carga masiva de elementos
  - **Te recomiendo crear un nuevo discussion para que alguien más del equipo pueda ayudarte. Normalmente es algo que hacemos por API**

**Reply by z0rro2k** [reply]
en eso estoy, no habia usado github antes, asi que aprendiendo de a poco (lo mismo con cotalker)
he estado revisando como hacer funcionar las api via curl o postman, aun no funciona ... pero funcionara
gracias

**Reply by acedward** [reply]
Un ejemplo de como activar una propiedad con CURL
```
curl 'https://www.cotalker.com/api/v2/properties/60fad97526a40700088b6d84' \
  -X 'PATCH' \
  -H 'Admin: true' \
  -H 'Content-Type: application/json' \
  -H 'Authorization: Bearer [token-secreto]' \
  -d '{"isActive":true}'
```

**Comment by agallardol** [comment]
Añadí el issue #405 para que podamos dalre seguimiento. Muchísimas gracias!
